src/Microsoft.Health.Dicom.Core/Features/Validation/DicomElementMinimumValidation.cs

```diff
@@ -1,223 +0,0 @@
-﻿// -------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-// -------------------------------------------------------------------------------------------------
-
-using System;
-using System.Globalization;
-using System.Linq;
-using System.Text.RegularExpressions;
-using Dicom;
-using Dicom.IO.Buffer;
-using Microsoft.Health.Dicom.Core.Exceptions;
-
-namespace Microsoft.Health.Dicom.Core.Features.Validation
-{
-    public static class DicomElementMinimumValidation
-    {
-        private static readonly Regex ValidIdentifierCharactersFormat = new Regex("^[0-9\\.]*$", RegexOptions.Compiled);
-        private const string DateFormatDA = "yyyyMMdd";
-        private const string BinaryDataPlaceHolder = "<BinaryData>";
-
-        internal static void ValidateAE(string value, string name)
```

Comment:
I like the generalization, but we still need type specific validation calling those.

Reply:
resolve as we discussed